rllib/core/learner/learner.py

```diff
@@ -540,7 +540,7 @@ def postprocess_gradients_for_module(
         """
         postprocessed_grads = {}
 
-        if config.grad_clip is None:
+        if config.grad_clip is None or not config.log_gradients:
```

Comment:
This is wrong, correct? So if someone switches off `config.log_gradients`, but still wants to grad-clip, then we DON'T grad-clip anymore :(

Reply:
I think it has to be `and`.

Reply:
Great catch! It should :)